src/client/pages/LoginApp/index.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import { Market, useMarket } from 'components/utils/CurrentLocale'
+import { LoadingPage } from 'components/LoadingPage'
+
+const LoginAppScreen = React.lazy(() =>
+  import(/* webpackChunkName: "debugger" */ './LoginAppScreen').then((m) => ({
+    default: m.LoginAppScreen,
+  })),
+)
```

Comment:
Just a question since I'm not very familiar with `React.lazy` or `Suspense` - why do we need to lazy load this particular component? Will this make this component not part of the main bundle or something? It's a pretty small component so I guess I don't really see why

Reply:
You're absolutely right, this will not make it part of the bundle. I did this to not load the LoginScreen for non Norwegian markets instead of rendering the component and showing an empty component. And I think we can enable code splitting more in the app, even if this small component itself won't make a huge difference :)

Reply:
I kind of don't really agree 😄 I think all things that brings more complexity to the codebase, like code splitting, should only be done when it's _really needed_. And it doesn't seem like it is in this case. I would suggest a simpler way, like:
```javascript
return market === Market.No ? <LoginAppScreen /> : null
```
Or is it possible to just say in the routes file that this particular route should only be available for the Norwegian market?

Reply:
That said, this is not a blocker since I guess it's pretty urgent to get this to prod. But as a rule of thumb I don't think we should make things more complex than they need to be (and I have no idea if we would actually _really need_ to do code splitting somewhere else in the repo, I mean maybe, but then we should have that as a joint strategy)!

Reply:
I’m all for not making things more complicated 😁 That being said I don’t necessarily think that code splitting makes it more complicated. But let’s remove it here and we can discuss when and what to use it for 🙌🏻 I’d like to keep the routes file clean without too many conditions so let’s do it like you suggested then 👍

Reply:
Awesome! 🙌  And it's a good point to keep the routes file as clean as possible 👍
And I'd love to learn more about code splitting and see where it could be useful for us!